packages/app-mobile/jest.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+	preset: 'react-native',
+
+	testMatch: ['**/*.test.js'],
```

Comment:
It might make sense to use `ts-jest` for tests (I remember running into a type-related testing issue, but don't remember what it was... Perhaps it was because VSCode's Jest extension works best if the `.ts` files are used directly...)

See also:
 * https://reactnative.dev/blog/2018/05/07/using-typescript-with-react-native#adding-typescript-testing-infrastructure
 * https://github.com/laurent22/joplin/pull/6650/files#diff-c880a5ff48742e618045f29337622542edb0b94b6c8579c8b9df86a9e6bb02cb

Reply:
Thanks for this. I'll look through it.

Reply:
@laurent22 Is it okay to use Typescript directly for testing?

Reply:
Eventually we might do that, but for now let's keep using the compiled .js files for testing.

Reply:
Wait, actually @personalizedrefrigerator set it up with ts-jest, is that right? In that case, yes please use this for this particular package.

Reply:
I'm just seeing this now.